src/coreclr/src/pal/src/libunwind/src/cross-dac/cross-dac.c

```diff
@@ -0,0 +1,97 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This is minimal implementation of posix functions files required to cross compile
+// libunwind on a Windows host for DAC application.
+
+#include "cross-dac.h"
+#include <stdlib.h>
+
+int getpagesize(void)
+{
+    // 4096 is truth for most targets
+    // Unlikely to matter in dump debugging
+    return 4096;
+}
+
+void* mmap(void *addr, size_t length, int prot, int flags, int fd, size_t offset)
+{
+    // We shouldn't be doing anything other than anonymous mappings
+    if ((flags & MAP_ANON) == 0)
+        return MAP_FAILED;
+
+    return malloc(length);
```

Comment:
`MAP_ANONYMOUS` guarantees a zero-initialized buffer, shouldn't we call memset on the buffer that was malloc'ed?